dockerfiles/util/keygen.Dockerfile

```diff
@@ -0,0 +1,38 @@
+# /*
+# |    Protect your secrets, protect your sensitive data.
+# :    Explore VMware Secrets Manager docs at https://vsecm.com/
+# </
+# <>/  keep your secrets… secret
+# >/
+# <>/' Copyright 2023–present VMware, Inc.
+# >/'  SPDX-License-Identifier: BSD-2-Clause
+# */
+
+# builder image
+FROM golang:1.20.1-alpine3.17 as builder
+
+RUN mkdir /build
+COPY app /build/app
+COPY core /build/core
+COPY vendor /build/vendor
+COPY go.mod /build/go.mod
+WORKDIR /build
+RUN CGO_ENABLED=0 GOOS=linux go build -mod vendor -a -o vsecm-keygen ./app/keygen/cmd/main.go
+
+# generate clean, final image for end users
+FROM gcr.io/distroless/static-debian11
```

Comment:
Since this is a utility module, we don’t need to create the usual, photon, fips, etc variants. — A distroless base is just fine.